src/main/scala/com/advancedtelematic/campaigner/db/Repository.scala

```diff
@@ -97,7 +99,7 @@ protected [db] class DeviceUpdateRepository()(implicit db: Database, ec: Executi
       .map(_.status)
       .update(status)
       .flatMap {
-        case 0 => DBIO.failed(Errors.DeviceNotScheduled)
+        case 0 => DBIO.failed(DeviceNotScheduled)
```

Comment:
Let's avoid prettying/formatting/whitespace changes in `feat`/`bugfix` PRs. Better to have separate `refactor` branches.